src/_P093_MitsubishiAC.ino

```diff
@@ -39,7 +39,7 @@ struct P092_data_struct : public PluginTaskData_base {
     setState(Connecting);
   }
 
-  virtual ~P092_data_struct() {
+  virtual ~P093_data_struct() {
     delete _serial;
```

Comment:
It looks like you always initialize the `_serial` object, but just to make sure it is always good practice to check if a pointer is zero before calling delete.
After all, a call to `new` can fail and then it will return a nullptr.

Reply:
Isn't deleting a null pointer safe and valid? https://isocpp.org/wiki/faq/freestore-mgmt#delete-handles-null

Reply:
Ah OK, wasn't aware of that. Thanks for the link.